requirements.txt

```diff
@@ -6,6 +6,7 @@ pyvisfile
 pymetis
 importlib-resources
 psutil
+gmsh
```

Comment:
This cannot stay. Installing `gmsh` via pip is not an option for us. It won't work on Lassen, and it may clash with our conda packages.

Reply:
I tried to replace this with a gmsh entry in `conda-env.yml`, but I was having some trouble getting it to work on lassen (I was getting dependency conflicts with cantera). @matthiasdiener mentioned that this might go away with conda-forge/gmsh-feedstock#44.